changelog/README.md

```diff
@@ -4,6 +4,7 @@
 
 ### 4.0.0 (in progress)
 
+- [improvement] JAVA-2182: QueryBuilder is missing json() method that takes object
```

Comment:
Could you describe what the commit _does_, instead of what's missing/not working before the commit? For example:
> Add insertInto().json() variant that takes an object in QueryBuilder

I know I'm being picky, and here it makes almost no difference, but for more complex changes I find that this approach produces better messages.